automation_testing/sv-mosaic.config.ts

```diff
@@ -0,0 +1,32 @@
+import { PlaywrightTestConfig } from "@playwright/test";
+
+const config: PlaywrightTestConfig = {
+
+    testDir: 'tests',
+    use: {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 1 tab but found 4 spaces.](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9498610)
```suggestion
	use: {
```